persistence/src/main/scala/za/co/absa/spline/persistence/tx/AbstractTxBuilder.scala

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
@wajda I would recommend reorganizing the number of subfolders, due to the complexity and avoiding the Doorway Effect

Reply:
Sure! Would you suggest anything specific? :)